### PR TITLE
webhook from ui design change

### LINKED ISSUE #6414: [CAL-755] Editing/Adding Webhooks UI Design - Layouts, spacing & type issues.
A lot of issues, particularly around the ping test area where font sizes are all over the place.

**Currently:**

**Should be**

[Settings/ Developer/ Webhooks/Add - Cal - Live (Figma)](https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=26%3A59917&t=3hNynQ6hBKZ73ols-1)

I'll add more spec shortly but if anyone can see the issues by comparing with the designs feel free to take on.

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-755](https://linear.app/calcom/issue/CAL-755/editingadding-webhooks-ui-design-layouts-spacing-and-type-issues)</sub>

### PR DESCRIPTION
**Pull request by joe-shajan**
## What does this PR do?

Fixes #6414

 This pull request is the same as #6647 I closed it and opened this PR. And all suggested changes were made here.

These are the changes I made.

image

In the Figma file, the font weight and colour look different visually but the values are the same. I need more clarification about wether I should change it or not.

And also all the other colours are also the same.

Currently, this is the heading, but in Figma the back arrow is outside. But when I try to change it, it will affect other places also. So should I go with it or keep it as it is?

image

## Type of change

- [x] UI update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: packages/features/webhooks/components/WebhookForm.tsx [hunk 1]
```diff
@@ -233,6 +233,7 @@ const WebhookForm = (props: {
                     { value: "default", label: t("default") },
                     { value: "custom", label: t("custom") },
                   ]}
+                  isFullWidth={true}
```
Comment (CarinaWolli) [inline packages/features/webhooks/components/WebhookForm.tsx:236]: design issue in mobile view
Screenshot 2023-01-31 at 10 37 38

File: packages/features/webhooks/components/WebhookTestDisclosure.tsx [hunk 2]
```diff
@@ -35,12 +31,24 @@ export default function WebhookTestDisclosure() {
           {t("ping_test")}
         </Button>
       </div>
-      <div className="p-4">
-        {!mutation.data && <em>{t("no_data_yet")}</em>}
-        {mutation.status === "success" && (
-          <div className="overflow-x-auto  text-gray-900">{JSON.stringify(mutation.data, null, 4)}</div>
-        )}
+      <div className="space-y-0 rounded-md border-0 border-neutral-200 sm:mx-0 md:border">
```
Comment (CarinaWolli) [inline packages/features/webhooks/components/WebhookTestDisclosure.tsx:34]: border missing in mobile view 
Screenshot 2023-01-31 at 10 38 46

File: packages/features/webhooks/components/WebhookTestDisclosure.tsx [hunk 3]
```diff
@@ -16,15 +16,11 @@ export default function WebhookTestDisclosure() {
...
-          {mutation.data && (
-            <Badge variant={mutation.data.ok ? "green" : "red"}>
-              {mutation.data.ok ? t("passed") : t("failed")}
-            </Badge>
-          )}
+    <>
+      <div className="flex justify-between">
+        <div>
+          <p className="text-sm font-semibold leading-5">{t("webhook_test")}</p>
+          <p className="font-sm mb-4 text-gray-600">{t("test_webhook")}</p>
```
Comment (CarinaWolli) [inline packages/features/webhooks/components/WebhookTestDisclosure.tsx:23]:
```suggestion
          <p className="text-sm mb-4 text-gray-600">{t("test_webhook")}</p>
```

**Review by CarinaWolli — DISMISSED** [review]
Looks good on desktop version already 🙏 but there are some issues in the mobile view

File: packages/features/webhooks/components/WebhookForm.tsx (hunk 1 above)
Comment (joeshajan) [reply packages/features/webhooks/components/WebhookForm.tsx:236]: image

It's working on my end. Do you know what the issue will be?

Comment (joeshajan) [reply packages/features/webhooks/components/WebhookForm.tsx:236]: And what about the ping test button? Should we keep it like that?

File: packages/features/webhooks/components/WebhookTestDisclosure.tsx (hunk 2 above)
Comment (joeshajan) [reply packages/features/webhooks/components/WebhookTestDisclosure.tsx]: done

File: packages/features/webhooks/components/WebhookTestDisclosure.tsx (hunk 3 above)
Comment (joeshajan) [reply packages/features/webhooks/components/WebhookTestDisclosure.tsx]: done

File: packages/features/webhooks/components/WebhookForm.tsx (hunk 1 above)
Comment (CarinaWolli) [reply packages/features/webhooks/components/WebhookForm.tsx:236]: > It's ok on my end. Do you know what the issue will be?

Try a bigger screen (Ipad), then you should be able to see the issue 

Also this doesn't look right in mobile for me (looks fine on your screenshot tho, did you change anything after?):
Screenshot 2023-01-31 at 17 32 45

Comment (CarinaWolli) [reply packages/features/webhooks/components/WebhookForm.tsx:236]: > And what about the ping test button? Should we keep it like that?

Yes, let's keep it like that

Comment (joeshajan) [reply packages/features/webhooks/components/WebhookForm.tsx:236]: > > It's ok on my end. Do you know what the issue will be?
> 
> Try a bigger screen (Ipad), then you should be able to see the issue
> 
> Also this doesn't look right in mobile for me (looks fine on your screenshot tho, did you change anything after?): Screenshot 2023-01-31 at 17 32 45

No I didn't change anything.

**Review by CarinaWolli — APPROVED** [review]
Tested again and everything looks good to me now, thank you @joeshajan 🙏

The event trigger select UI still needs to be fixed, but we can do that in a separate PR. Might be a general issue with our multi Select component

**Comment by joeshajan** [comment]
Thanks @CarinaWolli

**Pull request merged**